server/src/main/java/au/org/aodn/ogcapi/server/core/parser/AfterImpl.java

```diff
@@ -0,0 +1,72 @@
+package au.org.aodn.ogcapi.server.core.parser;
+
+import au.org.aodn.ogcapi.server.core.model.enumeration.StacExtent;
+import co.elastic.clients.json.JsonData;
+import org.geotools.filter.AttributeExpressionImpl;
+import org.geotools.filter.LiteralExpressionImpl;
+import org.opengis.filter.FilterVisitor;
+import org.opengis.filter.expression.Expression;
+import org.opengis.filter.temporal.After;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import co.elastic.clients.elasticsearch._types.query_dsl.*;
+
+import java.text.SimpleDateFormat;
+
+public class AfterImpl<T extends Enum<T>> extends ElasticFilter implements After {
+    protected Logger logger = LoggerFactory.getLogger(IntersectsImpl.class);
+
+    protected Expression expression1;
+    protected Expression expression2;
+
+
+    public AfterImpl(Expression expression1, Expression expression2, Class<T> enumType) {
+
+        this.expression1 = expression1;
+        this.expression2 = expression2;
+
+        if(expression1 instanceof AttributeExpressionImpl attribute && expression2 instanceof LiteralExpressionImpl literal) {
+            try {
+                SimpleDateFormat dateFormatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
```

Comment:
Once instance should be enough, no need to have it in the function level, I think it is better to be in ElasticFilter so it can be share with the DuringImpl which need exactly the same object